docs/content/developer/iota-move-ctf/challenge_1.mdx

```diff
@@ -0,0 +1,41 @@
+
+import ChallengeVerifier  from '@site/src/components/CTF/ctf-verifier';
+
+
+
+
```

Comment:
Can we get rid of the extra space used here? WDYT?

Reply:
```suggestion
```